### PR TITLE
Basic Generate new key pair flow

### PR DESCRIPTION
**Pull request by quietbits**

### CODE REVIEW

File: src/components/NewKeyPairForm.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+                being stolen.
+              </li>
+              <li>Only use this option if you're aware of the risks...</li>
+              <li>
+                Ideally use other authentication methods like a hardware wallet
+                or a browser
+              </li>
+              <li>...</li>
+            </ul>
+          </WarningEl>
```
Comment (morleyzhi) [inline src/components/NewKeyPairForm.tsx:118]: This warning doesn't seem to apply to this page? The user's not going to paste a key into this form.

File: src/components/NewKeyPairForm.tsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+      {acceptedWarning && (
+        <div>
+          <h2>New key pair</h2>
+
+          <WarningEl>
+            <h3>ATTENTION:</h3>
+
+            <ul>
+              <li>
+                It's really important to keep track of your secret key, and to
```
Comment (morleyzhi) [inline src/components/NewKeyPairForm.tsx:135]: You should get used to typing the right typographical character; that way, you don't have to painstakingly go back and fix them later. See also https://github.com/stellar/product-conventions/blob/master/STYLEGUIDE.md#use-the-right-symbols

File: src/components/NewKeyPairForm.tsx (hunk 1 above)
Comment (quietbits) [reply src/components/NewKeyPairForm.tsx:118]: I'm assuming this is just a placeholder warning text for now in wireframes.

File: src/pages/Landing.tsx [hunk 3]
```diff
@@ -3,15 +3,29 @@ import { Link } from "react-router-dom";
...
+const TempLinkButtonEl = styled.div`
+  margin-bottom: 20px;
+  text-decoration: underline;
+  cursor: pointer;
+`;
+
 export const Landing = () => {
-  const [modalVisible1, setModalVisible1] = useState(false);
-  const [modalVisible2, setModalVisible2] = useState(false);
+  const [newKeypaidModalVisible, setNewKeypaidModalVisible] = useState(false);
```
Comment (acharb) [inline src/pages/Landing.tsx:20]: Typo here :)  "keypaid"

Comment (quietbits) [reply src/pages/Landing.tsx:20]: 🤦‍♀️  Thanks!

**Pull request merged**